### ISSUE TITLE
Normalization 

### ISSUE THREAD

**Issue by Sidney2408** [abedicodes/ResNet-TCN#4]
Hi Abedi,

thank you for fielding most of my questions thus far. 

I was wondering, for transforms.py, why this particular normalization was used for image preprocessing
>> torchvision.transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]),

**Comment by abediai** [comment]
Hi,

Please see [this PyTorch documentation](https://pytorch.org/vision/stable/models.html):

"All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. You can use the following transform to normalize:"
